src/pages/core/guides/javascript/how-to-create-a-core-nft-asset.md

```diff
@@ -0,0 +1,378 @@
+---
+title: How to Create a Core NFT Asset
+metaTitle: How to Create a Core NFT Asset
+description: Learn how to create a Core NFT Asset on the Solana blockchain with the Metaplex Core package.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-18-2024'
+---
+
+This is an intial guide on how to create a next-gen Digital Asset Nft (Core Asset) on the Solana blockchain using the Metaplex Mpl Core protocol. Mpl Core Assets provide the next wave of NFT projects on Solana with greater creativity and a broad dynamic experiance for both creators and owners with Mpl Core's simplied design and unique plugin system.
```

Comment:
wdyt about:
"This is a guide on how to create a Core Asset, which is a next-gen Digital Asset on the solana blockchain using the Metaplex MPL Core protocol."

And then just calling them Core Assets everywhere else rather than trying to say "Core NFT Asset" or NFT Core Asset", which there's currently both formats used.

Reply:
I think I was just trying to get the NFT keyword into the page in logical places for SEO.